rfcs/0116-artifact-service.md

```diff
@@ -0,0 +1,196 @@
+# RFC 116 Artifact Service
```

Comment:
It's not an artifacts service... it's a generic blob/object storage service. Please use said terminology.